Resources/Views/SensimityView.xib

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="7706" systemVersion="14D136" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES">
+    <dependencies>
+        <deployment identifier="iOS"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="7703"/>
+    </dependencies>
+    <customFonts key="customFonts">
+        <mutableArray key="Geogrotesque-Light.otf">
+            <string>Geogrotesque-Light</string>
+            <string>Geogrotesque-Light</string>
+            <string>Geogrotesque-Light</string>
+            <string>Geogrotesque-Light</string>
+            <string>Geogrotesque-Light</string>
+        </mutableArray>
+        <mutableArray key="SourceSansPro-Regular.otf">
+            <string>SourceSansPro-Regular</string>
+            <string>SourceSansPro-Regular</string>
+            <string>SourceSansPro-Regular</string>
+        </mutableArray>
+    </customFonts>
+    <objects>
+        <placeholder placeholderIdentifier="IBFilesOwner" id="-1" userLabel="File's Owner"/>
+        <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
+        <view contentMode="scaleAspectFit" id="iN0-l3-epB" customClass="SensimityView">
+            <rect key="frame" x="0.0" y="0.0" width="316" height="488"/>
+            <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+            <subviews>
+                <imageView userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="sensimity" translatesAutoresizingMaskIntoConstraints="NO" id="wXw-Dm-93Z">
+                    <rect key="frame" x="20" y="35" width="276" height="50"/>
+                    <constraints>
+                        <constraint firstAttribute="height" constant="50" id="hF8-IR-roN"/>
+                    </constraints>
+                </imageView>
+                <textField opaque="NO" clipsSubviews="YES" tag="1" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" placeholder="Major id of advertising beacon" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="N8G-nR-5ql" customClass="SensimityTextField">
```

Comment:
Here `advertising` is causing a misunderstanding. Maybe it is better to use mock, dummy, ghost, etc instead.